Data/test.lua

```diff
@@ -0,0 +1,9646 @@
+      },
+      Formula="classlevel(\"APPLIEDAS=NONEPIC\")/3",
+      Conditions={
+        function (character)
+          return (character.Variables["UseAlternateSaveProgression"] == 0)
+        end,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "ClassBABModerate",
+      },
+      Formula="classlevel(\"APPLIEDAS=NONEPIC\")",
+      Conditions={
+        function (character)
+          return (character.Variables["UseFractionalBAB"] == 1)
+        end,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "BardLVL",
+      },
+      Formula="CL",
+    },
+    {
+      Category="VAR",
+      Variables={
+        "ClassSavePoor_Fortitude",
+      },
+      Formula="classlevel(\"APPLIEDAS=NONEPIC\")",
+      Conditions={
+        function (character)
+          return (character.Variables["UseFractionalSave"] == 1)
+        end,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "ClassSaveGood_Reflex",
+      },
+      Formula="classlevel(\"APPLIEDAS=NONEPIC\")",
+      Conditions={
+        function (character)
+          return (character.Variables["UseFractionalSave"] == 1)
+        end,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "ClassSaveGood_Will",
+      },
+      Formula="classlevel(\"APPLIEDAS=NONEPIC\")",
+      Conditions={
+        function (character)
+          return (character.Variables["UseFractionalSave"] == 1)
+        end,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "Caster_Level_BL_Stripped_Bard",
+      },
+      Formula="Caster_Level_Bard-CasterLevelBLBard",
+    },
+    {
+      Category="VAR",
+      Variables={
+        "Caster_Level_Bard",
+      },
+      Formula="CL+Caster_Level_Bonus+CasterLevelBLBard",
+    },
+    {
+      Category="VAR",
+      Variables={
+        "CasterLevelBL_x_Bard",
+      },
+      Formula="charbonusto(\"PCLEVEL\",\"Bard\")",
+    },
+    {
+      Category="VAR",
+      Variables={
+        "Caster_Level_Highest__Arcane",
+      },
+      Formula="Caster_Level_Bard",
+      Type={
+        Name="Base",
+        Replace=false,
+        Stack=false,
+      },
+    },
+    {
+      Category="VAR",
+      Variables={
+        "Caster_Level_Total__Arcane",
+      },
+      Formula="Caster_Level_Bard",
+    },
+    {
+      Category="CASTERLEVEL",
+      Variables={
+        "Bard",
+      },
+      Formula="Caster_Level_BL_Stripped_Bard",
+    },
+  },
+  WeaponBonusProficiencySelections={
+  },
+  SourcePage="p.34",
+})
+DefineClass({
+  HitDie=8,
+  MaxLevel=20,
+  SkillPointsPerLevel=2,
+  BonusLanguages={
+    {
+      Name="Abyssal",
+    },
+    {
+      Name="Celestial",
+    },
+    {
+      Name="Infernal",
+    },
+  },
+  Roles={
+    "Cleric",
+  },
+  Types={
+    "Base",
+    "PC",
+  },
+  Facts={
+    ClassType="PC",
+    Abb="Clr",
+    SpellType="Divine",
+  },
+  AutomaticKnownSpells={
+    "LstToLua.KnownSpell",
```

Comment:
This looks like object.ToString() to me.  Is something else supposed to be here?

Reply:
Yes, this is object.ToString(), I forgot an implementation here.